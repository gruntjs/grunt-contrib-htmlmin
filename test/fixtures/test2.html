<!doctype html>
<html>
  <body>
    test2
    <!-- Hello world -->
  </body>
</html>
