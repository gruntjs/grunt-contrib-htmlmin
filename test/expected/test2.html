<!doctype html><html><head><title>Test file 2</title></head><body>Hello world</body></html>